src/main/java/redis/clients/jedis/SentinelPoolConfig.java

```diff
@@ -0,0 +1,44 @@
+package redis.clients.jedis;
+
+import org.apache.commons.pool2.impl.GenericObjectPoolConfig;
+
+public class SentinelPoolConfig extends GenericObjectPoolConfig {
```

Comment:
GenericObjectPoolConfig takes a type argument. Declaring a sub-class without the type argument is anti-pattern.

Idea: Just declare this interface separately without sub-classing it; with a more appropriate name.

Reply:
may be  sub-class is the best way to avoid add more Constructor  method .  currently threse are 24  Constructor  method for JedisSentinelPool. 
use sub-class will not break the Constructor  parameter type GenericObjectPoolConfig.

Reply:
If that's your concern, you can introduce a Builder.